src/ORM/Table.php

```diff
@@ -1730,14 +1730,20 @@ protected function _processFindOrCreate($search, callable $callback = null, $opt
         }
         $entity = $this->newEntity();
         if ($options['defaults'] && is_array($search)) {
-            $entity->set($search, ['guard' => false]);
+            $entity = $this->patchEntity($entity, $search, ['guard' => false]);
```

Comment:
`guard` is not a valid option for `RepositoryInterface::patchEntity()` / `Marshaller::merge()`.  You need to use `accessibleFields` instead but requires specific field names as keys.